ocdata/vasp.py

```diff
@@ -31,6 +31,20 @@
               'pp': 'PBE',
               'xc': 'PBE'}
 
+# This is the setting for bulk optmization.
+# Only use when expanding the bulk_db with other crystal structures.
+BULK_VASP_FLAGS = {'ibrion': 1,
```

Comment:
This is currently unused in the code - did you want to just leave this here as a reference, or do we want to add a command line arg for users to optionally replace the lines of `vasp_flags = VASP_FLAGS.copy()` with `vasp_flags = BULK_VASP_FLAGS.copy()`? The latter would be cleaner, but I'm not sure if we'll be using this often enough to warrant changing command line args.

Reply:
Mostly want to leave it there as a reference. The use case would be input that as an arg to line 35 when we need it for bulk optimizations.  I agree it's not super clean as it is. I can move it to .CONSTANT folders if you think that's better?

Reply:
I'm OK leaving this here (or move both dicts to constants.py) as a reference for now. If we find ourselves needing to run with different flags often in the future, then we can move this to a command line arg.

Reply:
@janiceblue updated nsw, left bulk_vasp_flags in vasp.py for now until something changes. Ready to merge if everything else looks good to you.

Reply:
Sounds good, LGTM